sci-biology/discrover/discrover-1.5.2.ebuild

```diff
@@ -37,16 +37,19 @@ DEPEND="${RDEPEND}
 		dev-texlive/texlive-latexrecommended
 		media-gfx/imagemagick
 	)
+	lto? ( >=sys-devel/gcc-4.8 )
 "
 src_configure() {
 	local mycmakeargs=(
 		$(cmake-utils_use_with cairo CAIRO)
 		$(cmake-utils_use_with dreme DREME)
 		$(cmake-utils_use_with doc DOC)
+		$(cmake-utils_use_with lto LTO)
```

Comment:
I still use gcc 4.7 by default on my machine, also version 4.8 is installed. If I try to compile discrover with *ebuild discrover-1.5.2.ebuild compile* on my maschine it fails with:

> CMake Error at CMakeLists.txt:85 (MESSAGE):
>  Disabled: link-time optimization - LTO support in GCC version < 4.8 is
>  unreliable
>
>
>-- Configuring incomplete, errors occurred!

I think it's better to let the ebuild fail during pkg_pretend() like realized for [arrayfire](https://github.com/gentoo-science/sci/blob/master/sci-libs/arrayfire/arrayfire-9999.ebuild#L51). So if gcc < 4.8 is used **and** *lto* is set, the ebuild should directly quit the installation. But thats only my humble opinion ;)